etl/steps/data/garden/un/2024-07-25/refugee_data.meta.yml

```diff
@@ -0,0 +1,279 @@
+      - To calculate the number of refugees per 1,000 people in a country, we divide the number of refugees by the total population of the country and multiply by 1,000. We use our long term population data for this calculation.
+    display:
+      numDecimalPlaces: 0
+  description_international_protection: Other people in need of international protection refers to people who are outside their country or territory of origin, typically because they have been forcibly displaced across international borders, who have not been reported under other categories (asylum-seekers, refugees, people in refugee-like situations) but who likely need international protection, including protection against forced return, as well as access to basic services on a temporary or longer-term basis.
+  description_interally_displaced: Internally displaced persons (IDPs) are persons or groups of persons who have been forced or obliged to flee or to leave their homes or places of habitual residence, in particular as a result of, or in order to avoid the effects of armed conflict, situations of generalized violence, violations of human rights or natural or human-made disasters, and who have not crossed an internationally recognized State border. For the purposes of UNHCR’s statistics, this population includes only conflict-generated IDPs to whom the Office extends protection and/or assistance. The IDP population also includes people in an IDP-like situation.
+  description_stateless: |-
+    Individuals under UNHCR’s statelessness mandate are defined under the 1954 Convention Relating to the Status of Stateless People as those not considered as nationals by any State under the operation of its law. In other words, they do not possess the nationality of any State. UNHCR statistics refer to people who fall under the organization’s statelessness mandate as those who are stateless according to this international definition. Data from some countries may also include people with undetermined nationality. These are people who lack proof of possession of any nationality and at the same time have or are regarded as having important links to more than one State. UNHCR also works with populations at risk of statelessness, but persons at risk of statelessness are not reported on under the statistical category of individuals under UNHCR’s statelessness mandate.
+  description_asylum: |-
+    Persons who have filed an application for asylum in a country other than their own and whose claims have not yet been determined.
+    These include those filing primary applications or subsequent applications following an appeal. The date on which the application for asylum is filed marks their entry into the status of asylum seeker.
+    They remain in the status of asylum seeker until their application is considered and adjudicated.
+  description_refugee: |-
```

Comment:
This one doesn't clearly define what a refugee is. Instead it mentions the conventions where this is regulated.

Reply:
You can maybe use the dod definition

Reply:
These are the "descriptions from producer", which I have included to make clear what the UNHCR counts as a refugee (and where this is regulated). I can add the DOD text as a description_key additionally :)

Reply:
I think it's important to have a shorter definition of refugee somewhere, so description_key is a good option. It was weird for me to see that the producer doesn't define in more detail, though

Reply:
Added a description_key item with a general definition of refugees :)